frontend/app/src/store/balances/getters.ts

```diff
@@ -642,26 +642,25 @@ export const getters: Getters<
     return balances.map(value => value.label);
   },
 
-  assetInfo:
-    ({ supportedAssets, nonFungibleBalances }: BalanceState) =>
-    (identifier: string) => {
-      if (identifier.startsWith('_nft_')) {
-        for (const address in nonFungibleBalances) {
-          const nfb = nonFungibleBalances[address];
-          for (const balance of nfb) {
-            if (balance.id === identifier) {
-              return {
-                identifier: balance.id,
-                symbol: balance.name,
-                name: balance.name,
-                assetType: 'ethereum_token'
-              } as SupportedAsset;
-            }
+  assetInfo: (state: BalanceState) => (identifier: string) => {
```

Comment:
Was the problem that the destructured instance was on an old reference?

Reply:
Yes I guess. With destructuring, the supportedAssets will be `empty array` when searching.